CMSgov/mdct-mcr
# MDCT-2092: Button Verbiage Changes for AutoSave Experience

## Description

Closes [MDCT-2092](https://qmacbis.atlassian.net/browse/MDCT-2092). 

Change the "Save & continue" buttons to "Continue" on `StandardReportPage` components to match auto-save. Additionally, `ReportPageFooter` buttons are no longer type `Submit`. 

### How to test

Create a report and make sure the buttons are all say `Continue` now. Also, drawers and modals are unchanged. 

### Changed Dependencies

N/A

## Code author checklist

## Reviewer checklist (two different people)
